signin.html

```diff
@@ -0,0 +1,55 @@
+<html>
+
+<head>
+    <link rel="stylesheet" href="signin.css" />
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+
+<body>
+    <div id="MyModal" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" style=" color: white;"><i class="fa fa-sign-in" aria-hidden="true"></i>SIGN IN</h5>
+                </div>
+                <div class="modal-body">
+                    <h5>REGISTERED CUSTOMERS</h5>
+                    <hr />
+                    <form>
+                        <div class="form-group">
+                            <input type="username" class="form-control rounded-0" placeholder="Username*" formControlName="username">
+                        </div>
+                        <div class="alert">
+                            <!--<div>
+                                Please enter username
+                            </div> -->
+                        </div>
+                        <div class="form-group">
+                            <input type="password" class="form-control rounded-0" placeholder="Password*" formControlName="password">
+                        </div>
+                        <div class="alert">
+                            <!--<div>
+                                Please enter password
+                            </div> -->
+                        </div>
+                        <div class="authenticate col-md-12">
+                            <button type="button" class="Login btn  btn-block btn-lg col-md-12 rounded-0" (click)="login()">Login</button>
+                        </div>
+                    </form>
+                    <div class="form-group">
+                        <a href="" class="backLink">Back</a>

```

Comment:
Suggestion: provide `href` attribute some value if you don't want to give value provide it `#` value